src/graphics/axis.hpp

```diff
@@ -0,0 +1,95 @@
+#ifndef AXIS_HPP
+#define AXIS_HPP
+
+#include "drawable.hpp"
+#include "esthetic.hpp"
+
+#include <SFML/Graphics/CircleShape.hpp>
+#include <SFML/Graphics/RectangleShape.hpp>
+
+namespace graphic {
+
+class Axis : public Drawable {
+
+  private:
+    sf::RectangleShape line;
+    sf::CircleShape    arrow;
+
+    float        size_arrow;
+    sf::Vector2f size_line;
+    sf::Vector2f position_line;
+    sf::Vector2f position_arrow;
+    int          orientation;
```

Comment:
An enumeration would be clearer than an int. Should the orientation be possible at a given angle ? (I do not see a use case for it)

Reply:
I don't understand you cannot give to an enumeration a given value ? I mean I need sometimes orientation = 90.. How is this possible with enum ?

Reply:
If the axis has 4 possible directions (up, down, left and right) an enumeration is perfect. 
If we want a continuous angle (although I cannot think of any reason to have an axis at 42.34°) then a floating point variable makes more sense.

Reply:
In fact, I'm thinking about 3D.. there is a way that we need an continous control of orientation. What do you think ?